spec/lib/performify/initialize_spec.rb

```diff
@@ -25,12 +25,20 @@
       expect(subject.foo).to eq(args[:foo])
     end
 
+    it 'no filtered inputs' do
+      expect(subject.inputs).to be_nil
```

Comment:
Hm, I think that when we don't have validation `schema` for arguments, then maybe `inputs` should have all arguments? 
@kimrgrey @groony what do you think?

Reply:
@piton4eg I think it will be duplication of attribute `args `

Reply:
@piton4eg LGTM